src/WebJobs.Script/Binding/Http/ProxyHttpExtensions.cs

```diff
@@ -0,0 +1,391 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Binding
+{
+    internal static class ProxyHttpExtensions
```

Comment:
We already have Http/ JObject  serialization logic for Node.  Can we use that and avoid adding this?